sycl/test-e2e/NewOffloadDriver/aot-cpu.cpp

```diff
@@ -0,0 +1,6 @@
+// REQUIRES: opencl-aot, cpu
```

Comment:
we also need `any-device-is-cpu` i think

Reply:
I am not aware of this setting. But I can add that here. Again this test was lifted from ./sycl/test-e2e/DeviceCodeSplit/aot-cpu.cpp and I added --offload-new-driver